src/components/carousel/index.css

```diff
@@ -0,0 +1,80 @@
+.bg-img-container {
+  position: relative;
+  max-width: 100%;
+  height: 38rem;
+}
+
+.hero-bg-img {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  opacity: 0;
+  transition: opacity ease-in-out 0.4s;
+}
+
+.hero-bg-img-active {
+  opacity: 1;
+}
+
+/* .bg-img-container {
```

Comment:
If you do not need these css then it is better to remove those instead of commenting them

Reply:
Thanks for the review, I have commented those lines of code as I need them for future use.